crates/store/re_log_types/src/index/timestamp.rs

```diff
@@ -10,16 +10,16 @@ pub struct Timestamp(i64);
 impl Timestamp {
     #[inline]
     pub fn now() -> Self {
-        let ns_since_epoch = web_time::SystemTime::UNIX_EPOCH
+        let nanos_since_epoch = web_time::SystemTime::UNIX_EPOCH
             .elapsed()
             .expect("Expected system clock to be set to after 1970")
             .as_nanos() as _;
-        Self(ns_since_epoch)
+        Self(nanos_since_epoch)
     }
 
     #[inline]
-    pub fn from_ns_since_epoch(ns_since_epoch: i64) -> Self {
-        Self(ns_since_epoch)
+    pub fn from_ns_since_epoch(nanos_since_epoch: i64) -> Self {
```

Comment:
The getter was renamed to `nanos_since_epoch`, shouldn't this also be `from_nanos_since_epoch`?

Reply:
There's some re-namings left in the TODO on https://github.com/rerun-io/rerun/issues/8635